src/lib/y2network/connection_config/qeth.rb

```diff
@@ -23,6 +23,39 @@ module Y2Network
   module ConnectionConfig
     # Configuration for qeth connections
     class Qeth < Base
+      # For CCW devices and CCW group devices, this device ID is the device
```

Comment:
To be clear, the "read channel" is the "device ID" you are talking about, right?

Reply:
Well, the device ID is for both, the read channel and write channel. That is for example "0.0.8000" is the read channel device id and "0.0.8001" is the write channel device id.

https://www.ibm.com/support/knowledgecenter/linuxonibm/com.ibm.linux.z.lgdd/lxctcrot.jpg
https://github.com/SUSE/s390-tools/blob/master/ctc_configure#L10